content/posts/eks-vs-ecs.md

```diff
@@ -0,0 +1,107 @@
+
+
+### EKS Pros & Cons
+
+| Pros |
+| --- |
+| + All the power of Kubernetes |
+| + Any Kubernetes work done is portable to any other Kubernetes cluster |
+| + Easy to build tooling around |
+| + New integration with Fargate to add serverless functionality* |
+
+
+| Cons |
+| --- |
+| - Kubernetes can be challenging to learn |
+| - Without automation or tooling, the process of deploying a single application is more involved |
+| - Can be overkill for small projects |
+
+
+* It's important to note that with the Fargate integration, there are some limitations such as which regions you can deploy to [https://docs.aws.amazon.com/eks/latest/userguide/fargate.html](https://docs.aws.amazon.com/eks/latest/userguide/fargate.html).
+
+
+## Running CloudBees Core/Jenkins
+
+Now the chances are you are reading this to learn about which platform to use for CloudBees Core or Jenkins. As mentioned earlier, we have a preference for Kubernetes for a few of reasons which I will get into here.
+
+### Portability
+
+Kubernetes is an open-standard. When you build your CI/CD system on Kubernetes, it can run on EKS, GKE, AKS, OpenShift, PKS, home-built, etc. without issue (well some vendor flavors like OpenShift have additional resources which aren't portable to standard k8s clusters).
+
+Some organizations are fine with going all-in on a single cloud vendor, but many others try to avoid vendor lock-in as much as possible. Keeping to as many open and portable platforms as possible ensures that the effort you do while building your system won't be wasted if you switch from AWS to GCP or Azure.
+
+The ECS platform exists only in AWS and is proprietary, so you don't have portability to other vendors. 
+
+This is a big plus for using EKS (or any Kubernetes).
+
+
+### Community
+
+As of writing this post, Kubernetes is the open standard for container orchestration, in major part because of it's huge and rapid community adoption. Since it is the standard, a large amount of DevOps tooling has been built on top of it. Most CI/CD processes will run through many different tools, so opting for the platform with the greatest marketshare is a safe choice.
+
+Because of this being the reality, we at CloudBees have built lots of functionality into CloudBees Core specific to Kubernetes which adds a lot of quality of life improvements. The ability to dynamically provision masters and agents is in my opinion a game changer. There are features like hibernating masters which we have recently released as a technical preview which allows for potentially huge cost savings by spinning down masters that haven't been used in a while. Functionality like this is only possible on a consistent platform, and Kubernetes gives us this.
+
+
+## Wrapping it up
+
+When it comes to running CloudBees Core on a modern platform, Kubernetes is our recommendation to you if it is at all possible. With recent changes to Amazon's EKS to support Fargate, it is becoming even easier to manage your environment.
```

Comment:
I would remove "if it is at all possible" - anything is possible and we should be more firm with our opinions. 

Mentioning Fargate on EKS seems a bit misleading - at least in the context of Core or Jenkins since I don't believe it is possible to run either there yet.